db/migrations/20190503090026_add_maintenance_info_to_service_instance.rb

```diff
@@ -0,0 +1,5 @@
+Sequel.migration do
+  change do
+    add_column :service_instances, :maintenance_info, :text, null: true
```

Comment:
We have a policy in our [migration style guide](https://github.com/cloudfoundry/cloud_controller_ng/wiki/CAPI-Migration-Style-Guide#always-specify-a-size-for-a-string-and-dont-use-text) to not use text columns and to specify length of the string, something like:
```
add_column :service_instances, :maintenance_info, String, size: 4096, null: true
```
Feel free to replace the `4096` with a value appropriate to the application.